docs/my-website/docs/observability/opentelemetry_integration.md

```diff
@@ -0,0 +1,57 @@
+import Image from '@theme/IdealImage';
+
+# 🔭 OpenTelemetry - Tracing LLMs with any observability tool
```

Comment:
can you remove the emoji ` 🔭 `

Reply:
@ishaan-jaff but everyone else has an emoji I want one too! 😅